test/issues.js

```diff
@@ -21,36 +21,46 @@ var lib = require('../lib')
   , couch = require('./couch')
   , follow = require('../api')
 
+var getSeq = couch.get_update_seq; // alias
+
 couch.setup(test)
 
+
 test('Issue #5', function(t) {
-  var saw = { loops:0, seqs:{} }
-
-  var saw_change = false
-  // -2 means I want to see the last change.
-  var feed = follow({'db':couch.DB, since:-2}, function(er, change) {
-    t.equal(change.seq, 3, 'Got the latest change, 3')
-    t.false(saw_change, 'Only one callback run for since=-2 (assuming no subsequent change')
-    saw_change = true
-
-    process.nextTick(function() { feed.stop() })
-    feed.on('stop', function() {
-      // Test using since=-1 (AKA since="now").
-      follow({'db':couch.DB, since:'now'}, function(er, change) {
-        t.equal(change.seq, 4, 'Only get changes made after starting the feed')
-        t.equal(change.id, "You're in now, now", 'Got the subsequent change')
-
-        this.stop()
-        t.end()
-      })
+  // CouchDB 1.X only test
```

Comment:
It'd be good to know why this is a 1.x only test

Reply:
Add comment.